net-analyzer/ossec-hids/ossec-hids-3.1.0.ebuild

```diff
@@ -0,0 +1,82 @@
+# Copyright 1999-2018 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit user
+
+DESCRIPTION="Open Source Host-based Intrusion Detection System"
+HOMEPAGE="https://www.ossec.net/"
+SRC_URI="https://github.com/ossec/ossec-hids/archive/${PV}.tar.gz -> ${P}.tar.gz"
+LICENSE="GPL-2"
+SLOT="0"
+
+KEYWORDS="~amd64"
+
+IUSE="agent hybrid local mysql postgres server sqlite zlib"
+DEPEND="mysql? ( virtual/mysql )
+	sqlite? ( dev-db/sqlite:3 )
+	postgres? ( dev-db/postgresql:= )
+	zlib? ( sys-libs/zlib:= )"
+RDEPEND="${DEPEND}"
+
+S="${WORKDIR}/${P}/src"
+
+MY_PREFIX="/var/ossec"
+
+my_options() {
+	local target="local"
+	if use agent; then
+		target="agent"
+	elif use hybrid; then
+		target="hybrid"
+	elif use server; then
+		target="server"
+	fi
+
+	local opts="V=0 USE_SQLITE=$(usex sqlite) ZLIB_SYSTEM=$(usex zlib)"
+	if use mysql; then
```

Comment:
Again, looks like case for `^^`. Users don't expect `USE="mysql postgres"` means 'random of the two'.

Reply:
As mentioned above, I used [Conflicting USE Flags](
https://devmanual.gentoo.org/general-concepts/use-flags/index.html) as a guideline (emphasis as in original text):

> Occasionally, ebuilds will have conflicting USE flags for functionality. Checking for them and returning an error is _not_ a viable solution. Instead, you must pick one of the USE flags in conflict to favour and should alert the user that a particular flag is being used instead.

Reply:
Added REQUIRED_USE. I think this addresses the last open issue (as of now).